projects/core/components/scroll-controls/scrollbar-wrapper.directive.ts

```diff
@@ -0,0 +1,12 @@
+import {Directive, ElementRef, Inject} from '@angular/core';
+
+@Directive({
+    selector: '[tuiScrollbarWrapper]',
```

Comment:
Please remove `TUI_ELEMENT_REF` provider from `ScrollControlsComponent` and move it here. It is only used to get height for vertical and width for horizontal bars which should work the same if you use this directive instead.

Reply:
Yep, good idea, thanks! ) I've changed this